Graphics/ShaderTools/src/ShaderToolsCommon.cpp

```diff
@@ -157,12 +157,38 @@ void AppendShaderSourceCode(std::string& Source, const ShaderCreateInfo& ShaderC
     Source.append(SourceData.Source, SourceData.SourceLength);
 }
 
+static void GePositionInfo(const Char* pBuffer, size_t BufferSize, size_t Offset, size_t& Line, size_t& LineOffset)
```

Comment:
Separate function is not needed - it is very short and is used in a single place only